### ISSUE TITLE
Expand wiki documentation

### ISSUE THREAD

**Issue by vigji** [3d-pli/fastpli#6]
I have been reading through the wiki and I can give some feedback on unclear points/things that I think might help clarifying stuff for the user. I apologise in advance if I am saying stuff just because I missed pieces, I just want to give fully constructive feedback from a completely external eye! :) 

`fastPLI is an open source toolbox for modeling nerve fibers, simulating them in a 3D PLI [I would probably have it as Polarized light imaging at least in the package line description] microscope and --the-- [simulation?] signal processing.`

I take it is not the case, but I would make it even clearer if it is just for simulation, or also for PLI data analysis (`fastpli.analysis` is the first module you read about in the README, and you might wonder if you can use the package also to just analyse real world data you might provide).

Probably it would be good to guide the user through a reasonable reading order, either with a sequential list of readings in the home of the user that a person should go through, or a reasonable order of pages on the right. Eg, you might want to have them reading about how FiberBundles are represented as the first step. Moreover, you probably want in the organisation of such reading to split even more clearly everything that is related to the generation of the bundles (the sandbox, collision solving, etc); from the reading you would need to do if you just want to simulate the PLI results from the fibres.

I think that adding some intro lines to every page where you showcase code might be useful (as you do eg in https://github.com/3d-pli/fastpli/wiki/Simulation but not in https://github.com/3d-pli/fastpli/wiki/Crossing).

Finally, for users to start get hands in in the package usage, it would be great if those wiki pages could also be distributed as tutorial jupyter notebooks in the package!

**Comment by fmatuschke** [comment]
Thank you for your suggestions. Your external eye is much appreciated. This is after all the idea of a review :).

I am currently redesigning the Wiki pages. This will still take some time. It is a very good idea to guide the user more through the process of the toolbox.

I also like your idea of a jupyter notebook as an interactive tutorial. Currently, I added first drafts for the `sandbox` and `solver` process inside the `./examples` folder. Their output will also be available at the corresponding Wiki pages.

I will inform you as soon as I have an official release.

**Comment by vigji** [comment]
I'll put here a cumulative reply:
Great that the suggestions were appreciated, let me know whenever you think the next release incorporating these changes is ready, I am looking forward to have another look! :)

**Comment by fmatuschke** [comment]
I have redesigned my wiki. It took me some time, but I hope it is much clearer now.
I tried to highlight the three basic components of my software. In addition, as you suggested, I have written a tutorial for each part in a Jupyter notebook. These tutorials are available within the wiki as a file converted to Markdown. The same notebook can also be executed by the user from the actual ipynb file in `examples/`.
